zulipterminal/ui_tools/messages.py

```diff
@@ -176,7 +176,11 @@ def stream_header(self) -> Any:
     def private_header(self) -> Any:
         title_markup = (
             "header",
-            [("general_narrow", "You and "), ("general_narrow", self.recipients_names)],
+            [
+                ("general_narrow", " § "),
```

Comment:
It'd be good to add this new symbol in the centralized config/symbols.py.

Reply:
Right, I'll get on that.